contents/blog/alerts-examples.mdx

```diff
@@ -0,0 +1,193 @@
+
+## How to create alerts
+
+Alerts are based on your insights. You can set them with your existing [trends](/docs/product-analytics/trends/overview), or create a new one.
+
+To create an alert, click the **Alerts** button on the top right of the insight page.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/alerts_button_4ecd022cd6.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/alert_button_dark_c2f642fde3.png"
+    alt="alerts button"
+    classes="rounded"
+/>
+
+To configure the alert, set the following:
+
+- **Name:** The name you'll see when the alert is triggered.
+
+- **When:** The data points the alert will be based on. The drop-down contains a list of all the series in your insight's graph. For example, if you have an insight that tracks the number of pageviews and signups, you can use either of these series as the basis for the alert threshold.
+
+- **Type:** Select "has value" to check if the series is more or less than an absolute amount. Or, select "increases by/decreases by" to check if the series increased or decreased by a relative amount. These can also be configured with a percentage instead of a static value (e.g. signups increased by 10%).
+
+- **Frequency:** Set how often you want to check the alert threshold (hourly, daily, weekly, monthly).
+
+- **Who to notify:** Select who should receive an email when the alert is triggered.
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/filled_alert_627158e6bb.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/alert_people_dark_0ccd7611bd.png"
+    alt="filled alert"
+    classes="rounded"
+/>
+
+Then, click **Create alert** and you'll see the new alert:
+
+<ProductScreenshot
+    imageLight="https://res.cloudinary.com/dmukukwp6/image/upload/insight_alerts_5k_daef47ade1.png"
+    imageDark="https://res.cloudinary.com/dmukukwp6/image/upload/created_alert_dark_1710da74dc.png"
+    alt="insight alerts"
+    classes="rounded"
+/>
+
+For a more detailed walkthrough, you can also check out our [Alerts docs](/docs/alerts), which we'll update as we add new features.
+
+## Practical uses for alerts
+
+Now that we can create alerts, let's take a look at a handful of practical examples.
+
+### 1. Get alerts when traffic spikes
+
+Want to investigate spikes in traffic to your website? Monitoring the traffic to your website can help you measure the success of marketing campaigns and understand which activities drive the most visitors to your site.
+
+These kinds of alerts can also be useful when traffic spikes are unexpected. We often see this at PostHog when our content gets featured in a popular email newsletter or shared on Hacker News. When we notice an influx of new signups, it can be fun to dig in and try to figure out where they came from.
```

Comment:
I think: 

> Want to investigate spikes in traffic to your website? Monitoring the traffic to your website can help you measure the success of marketing campaigns and understand which activities drive the most visitors to your site.

is a use case for normal insights where traffic spikes is a good use case for alerts. 

```suggestion
Traffic spike alerts are useful when this happens unexpectedly. We often see this at PostHog when our content gets featured in a popular email newsletter or shared on Hacker News. When we notice an influx of new signups, it can be fun to dig in and try to figure out where they came from.
```